libtransmission/resume.cc

```diff
@@ -831,7 +831,7 @@ auto setFromCtor(tr_torrent* tor, tr_resume::fields_t fields, tr_ctor const* cto
     {
         if (auto is_paused = bool{}; tr_ctorGetPaused(ctor, mode, &is_paused))
         {
-            tor->isRunning = !is_paused;
+            tor->start_when_stable = !is_paused;
```

Comment:
This seems to always set `start_when_stable` to `false` when app is starting. So we lose the information regarding `start_when_stable` from previous session and are forced to either start or not all magnets, independently of user choice from previous session.

Reply:
Information seems to be lost due to `tr_ctorSetPaused(ctor, TR_FORCE, true)` in Controller.mm